go.mod

```diff
@@ -39,5 +36,8 @@ require (
 	github.com/json-iterator/go v0.0.0-20171115153421-f7279a603ede // indirect
 	github.com/mschoch/smat v0.2.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
-	golang.org/x/sys v0.5.0 // indirect
+	golang.org/x/exp v0.0.0-20231006140011-7918f672742d // indirect
+	golang.org/x/sys v0.13.0 // indirect
 )
+
+replace github.com/blevesearch/zapx/v16 => ../zapx
```

Comment:
You can remove this and use the branch `refactor-section` for zapx/v16 before running `go mod tidy` again. This is until we merge refactor-section into master.